CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Issues
+
+Issues have three types: bug, feature request, and support. When reporting a bug, you must include
+the operating system used, any relevant information about the tech stack, and the feature flags
+used. 
```

Comment:
I don't have any policy in mind for feature requests or support issues. Please suggest changes.

Reply:
> support issues

We should probably try redirecting people to the Discord for support and leave support issues as either a tracking issue or fallback if the person is opposed to using Discord. For the issue contents though, I think the operating system and feature flags are solid. We could also attempt to ask for a reproduction example to make Twilight dev's lives easier. The reporter must also provide whatever error rustc is giving them, if applicable. 

> feature requests

I'm thinking of a format like this:
```markdown
## What would you like implemented?
<Thing they want that Twilight lacks>

## What use cases does this request address?
<More then a single project's specialized use case>

## Are you willing to help towards contributing this feature?
Note: Its perfectly fine if you say no, but we encourage you to try :) 
<The submitter's answer> 

## What would the feature look like and other info
<Other snippets, examples, etc>
```

This would deflect the low effort "Give me xyz" issues every project dreads and, I think at least, sits on a nice sweet spot of detailed and too much work.